### DISCUSSION TITLE
Login in GitHub with phone number

### DISCUSSION THREAD

**Discussion by ghost** [community/community#18826] [Other Feature Feedback, Questions, & Ideas]
#### 1. feature-name
Login in GitHub with phone number

#### 2. feature-description
Login in GitHub with phone number

#### 2.1 concept
![image](https://user-images.githubusercontent.com/95204402/174417308-60705c28-48a0-42e4-912a-ab93495c70e2.png)

#### 3. Why?
1. This would allow better login, usually known programs have many options to log into the system with username, email or phone number, as in the case of programs like Uber
2. Remembering passwords and email can usually be complicated for some people, logging in with the phone number you receive an sms message and you log in and that's it
3. Recently GitHub was purchased by Microsoft, this means or could mean now presently or in the future an integration with Microsoft services, for example, if we want to verify a user account, we can verify that the number associated with the account belongs to Skype or LinkedIn
4. It would be interesting to have a better integration of GitHub with LinkedIn in order to avoid any spam or fake accounts
5. This could increase reliability on GitHub

#### 4. Notes
I am not promoting or advertising any company, service, person, technology or product here - all links added are bibliographic references.

#### 5. References
- https://www.uber.com/br/en/

**Comment by doguedogue** [comment]
Looks like Two-factor authentication 🚀🚀🚀

**Reply by ghost** [reply]
@doguedogue Hi! thank you for feedback ;D

**Comment by ghost** [comment]
I liked this proposal.
